packages/qwik/src/optimizer/src/plugins/vite-server.ts

```diff
@@ -118,7 +118,7 @@ export async function configureDevServer(
                   location: 'head',
                   attributes: {
                     rel: 'stylesheet',
-                    href: url,
+                    href: `${opts.base}${url}`,
```

Comment:
Isn't it possible that done of these urls already include the base?

Better to change them at the source, no?